docs/workflows/sdk-setup.mdx

```diff
@@ -22,277 +66,324 @@ The Embedded SDK is a holistic SDK solution offering the entire experience embed
+</TabItem>
+</Tabs>
+
+## Setup
 
-#### From Xcode
+<Tabs groupId="sdks" queryString>
+<TabItem value="javascript" label="JavaScript">
+
+<SetupJavaScript/>
 
-1. From the Xcode `File` menu, select `Add Packages` and add the following url:
+</TabItem>
+<TabItem value="kotlin" label="Kotlin">
 
-```
-https://github.com/gobeyondidentity/bi-sdk-swift
-```
+<SetupKotlin/>
 
-2. Select a version and hit Next.
-3. Select a target matching the SDK you wish to use.
+</TabItem>
+<TabItem value="swift" label="Swift">
 
-#### From Package.swift
+<SetupSwift/>
 
-1. With [Swift Package Manager](https://swift.org/package-manager),
-   add the following `dependency` to your `Package.swift`:
+</TabItem>
+<TabItem value="reactnative" label="React Native">
 
-```javascript
-dependencies: [
-    .package(url: "https://github.com/gobeyondidentity/bi-sdk-swift.git", from: [version])
-]
-```
+<SetupReactNative/>
 
-2. Run `swift build`
+</TabItem>
+<TabItem value="flutter" label="Flutter">
 
-### Cocoapods
+<SetupFlutter/>
 
-Add the pod to your Podfile:
+</TabItem>
+</Tabs>
 
-```javascript
-pod 'BeyondIdentityEmbedded'
-```
+## Next Steps
 
-And then run:
-```javascript
-pod install
-```
+<Tabs groupId="sdks" queryString>
+<TabItem value="javascript" label="JavaScript">
 
-After installing import with
-```javascript
-import BeyondIdentityEmbedded
-```
+<!---
+- [Bind a Passkey](./bind-passkey)
+- [Authentication](./authentication)
+-->
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-js/)
 
 </TabItem>
-<TabItem value="reactnative" label="React Native">
+<TabItem value="kotlin" label="Kotlin">
+
+<!---
+- [Bind a Passkey](./bind-passkey)
+- [Authentication](./authentication)
+-->
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-android/)
 
-```javascript
-yarn add @beyondidentity/embedded-react-native
-```
+</TabItem>
+<TabItem value="swift" label="Swift">
 
-or
+<!---
+- [Bind a Passkey](./bind-passkey)
+- [Authentication](./authentication)
+-->
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-swift/documentation/beyondidentityembedded/)
 
-```javascript
-npm i --save @beyondidentity/embedded-react-native
-```
+</TabItem>
+<TabItem value="reactnative" label="React Native">
 
-:::warning
-Note that this library is not compatible with Expo projects. This is because Expo's “managed” workflow does not allow usage of React Native libraries that introduce their own native code beyond the React Native APIs and components that are available in the Expo client app. If you have an Expo app and wish to use this project, you must eject.
-:::
+<!---
+- [Bind a Passkey](./bind-passkey)
+- [Authentication](./authentication)
+-->
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-react-native/)
 
 </TabItem>
 <TabItem value="flutter" label="Flutter">
 
-### Pub.Dev
+<!---
+- [Bind a Passkey](./bind-passkey)
+- [Authentication](./authentication)
+-->
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-flutter/)
```

Comment:
I wonder if this should be its own section like `Sample App` at the top? Maybe the section could be called something like `SDK API Documentation`

Reply:
Do you mean move this section underneath "Sample App" and rename it to "SDK API Documentation"? I did consider it there, but I thought it would be better to walk developers through the basic setup before showing them all of the other functions. What do you think?

I do like "SDK API Documentation" as a header better.

Reply:
I was thinking it could have it's own section like how "Sample App" has it's own section and header.

Reply:
Oh I see you changed it! Yea like that but with the icons too maybe and some text like the sample app section